### PR TITLE
Italian translations for vue of intro-to-storybook

### PR DESCRIPTION
**Pull request by Archetipo95**
Huge thanks to @ludovicobesana for the React translations which I took for this Vue translation

### CODE REVIEW

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@Archetipo95, thank you so much for taking the time to put together this pull request and helping our community get a better understanding of Storybook by extending our translations. We truly appreciate it 🙏 !

I left a couple of small items for you to look into when you have a moment.

Also, @ludovicobesana, when you have a moment, can you take a pass at this pull request from a language standpoint so we can polish it and release it?

Hope both of you have a great day.

Stay safe.

**Comment by Archetipo95** [comment]
@jonniebigodes I fixed the issues that you mentioned in the comments. If you have something else feel free to give me a feedback!

**Comment by jonniebigodes** [comment]
@Archetipo95 appreciate you addressing the feedback so promptly 🙏 ! From my end, all is good. I'll message @ludovicobesana tomorrow morning my time and see if he has time to take a look into this, so that we're able to get the feedback addressed as soon as possible and publish the translation.

If he's unable to review it in a reasonable timeframe. I'll go ahead and merge this, and we can make some adjustments in a follow-up pull request.

Sounds reasonable to you?

Have a fantastic day.

Stay safe

**Comment by Archetipo95** [comment]
Absolutely, it sounds awsome! Addressing feedback promptly is crucial for maintaining momentum 🚀

Wishing you a fantastic day as well!

**Comment by ludovicobesana** [comment]
@jonniebigodes I'll do the review by Sunday 7th.
Thanks again for your contribution @Archetipo95 🙏🏻

**Comment by jonniebigodes** [comment]
Appreciate it @ludovicobesana 🙏

**Review by jonniebigodes — APPROVED** [review]

**Comment by ludovicobesana** [comment]
@Archetipo95 I'm looking it right now in code from here: https://github.com/Archetipo95/learnstorybook.com/tree/master

File: content/intro-to-storybook/vue/it/get-started.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+tocTitle: 'Inizia'
+description: 'Configura Vue Storybook nel tuo ambiente di sviluppo'
+commit: 'e2984d1'
+---
+
+Storybook funziona in parallelo con la tua app in modalità di sviluppo. Ti aiuta a costruire componenti dell'interfaccia utente isolati dalla logica aziendale e dal contesto della tua app. Questa edizione del tutorial Intro to Storybook è per Vue; esistono altre edizioni per [React](/intro-to-storybook/react/it/get-started), [React Native](/intro-to-storybook/react-native/en/get-started/), [Angular](/intro-to-storybook/angular/en/get-started), [Svelte](/intro-to-storybook/svelte/en/get-started) ed [Ember](/intro-to-storybook/ember/en/get-started).
+
+![Storybook e la tua app](/intro-to-storybook/storybook-relationship.jpg)
+
+## Configura Storybook per React
```
Comment (ludovicobesana) [inline content/intro-to-storybook/vue/it/get-started.md:12]: @Archetipo95 Qui "Configura Storybook per Vue"

**Review by ludovicobesana — CHANGES_REQUESTED** [review]

**Comment by ludovicobesana** [comment]
@Archetipo95 I've done a quick review, could you please check the feedbacks that I've added?
Thank you! 🙏🏻

Now I need to go but tomorrow I'll check if I found other things to improve

**Comment by Archetipo95** [comment]
@jonniebigodes I resolved all the suggestions that @ludovicobesana raised. If you want to close it feel free :D

**Comment by Archetipo95** [comment]
Hi :D @ludovicobesana I did both fixes in one commit
Am I missing something?
Screenshot 2024-04-15 at 22 05 10

**Review by ludovicobesana — CHANGES_REQUESTED** [review]
@Archetipo95

**Comment by ludovicobesana** [comment]
> Hi :D @ludovicobesana I did both fixes in one commit Am I missing something? Screenshot 2024-04-15 at 22 05 10

@Archetipo95 I've re-added this points and also if you can check also the other new parts of translations that could be great!

**Comment by Archetipo95** [comment]
Done! Sorry I messed something and I was not seeing your suggestions :D thanks for tagging me @ludovicobesana 
Now everything should be resolved, I accepted all of your suggestions, thanks for taking time to check this PR

**Comment by ludovicobesana** [comment]
> Done! Sorry I messed something and I was not seeing your suggestions :D thanks for tagging me @ludovicobesana Now everything should be resolved, I accepted all of your suggestions, thanks for taking time to check this PR

Cool thank you, did you check the other documentation files too?
Just to have a double check.

**Comment by jonniebigodes** [comment]
@Archetipo95 and @ludovicobesana can you give me a status update on this pull request so that we can continue to work on it or get it out? 

Let me know and we'll go from there.

Have a fantastic day the both of you.

**Comment by Archetipo95** [comment]
@jonniebigodes I did the suggestions that @ludovicobesana  made and I'm done with this Pull Request for now. :)

**Comment by ludovicobesana** [comment]
Hi @jonniebigodes if @Archetipo95 has resolved all the points and reviewed the other files I think that we can merge it 🚀

**Comment by jonniebigodes** [comment]
@Archetipo95, I appreciate you following up with me on this, and I will merge this so we can get it published.

One small thing to take care of, @Archetipo95. If you're ok with it, can you message me directly (same username) on our [Discord Server](https://discord.gg/storybook) so that I can follow up with you on this pull request?

@ludovicobesana cannot thank you enough for your time in getting this proofread 🙇 

Hope both of you have a great day.

Stay safe

**Pull request merged**